docs/content/Getting-Started/user-quickstart.md

```diff
@@ -0,0 +1,80 @@
+
+[![QuickStart test]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml/badge.svg?branch={{config.ks_branch}})]({{config.repo_url}}/actions/workflows/docs-ecutable-qs.yml)&nbsp;&nbsp;&nbsp;
+
+
+!!! tip "Estimated time to complete this example:" 
+    ~4 minutes (after installing prerequisites)
+
+### How to deploy and use KubeStellar
+
+This guide is intended to show how to (1) quickly bring up a **KubeStellar** environment using helm, (2) install plugins with brew, (3) retrieve your kubeconfig, (4) install KubeStellar Syncers on 2 edge clusters, (5) and deploy an example workload to both edge clusters.
+
+## 0. Pre-reqs
+
+helm
+brew
+kubectl
+3 kind clusters 
+   1 KubeStellar core cluster
+   2 KubeStellar edge clusters (edge-cluster1, edge-cluster2)
+   
```

Comment:
Maybe this is just a problem with the git preview, but the above does not format correctly for me.